index.html

```diff
@@ -49,6 +49,9 @@ <h1 class="title">Don't build (or build) that feature</h1>
                 <input type="range" min="1" max="10" step="0.5" value="5.5" />
             </fieldset>
         </form>
+        <div class="text-center">
+          <button id="answer-visibility-toggle"></button>
```

Comment:
This should have a default value for accessibility.

`<button id="answer-visibility-toggle">Show Answer</button>`

Reply:
Understood. Amending it